content/2_Bronze/Ad_Hoc.mdx

```diff
@@ -35,6 +36,286 @@ approaching problems that appear to be ad hoc.
 In the end, the best way to get better at ad hoc problems (or any type of
 problem) is to do a lot of them.
 
+### Example - Milking Order
+
+<FocusProblem problem="tutorial" />
+
+Don't be afraid to give up on some approach if you aren't making progress!
+
+<Spoiler title="Hint 1">
+Let's try placing cow $1$ at a specific position.
+
+How might we go about checking to see if we'll end up with a valid ordering?
+</Spoiler>
+
+<Spoiler title="Solution 1">
+[Official Analysis (C++)](http://www.usaco.org/current/data/sol_milkorder_bronze_open18.html)
```

Comment:
also idt this should be here
tbh you should probably rewrite some (if not all) of the thing since it's a module sol

Reply:
👍

will try to do by tomorrow